### DISCUSSION TITLE
how to connect to azure search service to query the index

### DISCUSSION THREAD

**Discussion by nanyu2017** [microsoft/autogen#4269] [Q&A]
autogen.net sdk could be used to connect to multiple agent like openai and gemini , just want to know how can i connect to azure ai search for query the content in index . Thanks in advance!

**Comment by rysweet** [comment]
You can find some examples for how to use Azure AI Search [here](https://learn.microsoft.com/en-us/azure/search/samples-python) 
so you could augment your autogen agents with some code that follows those examples.
